examples/tester.js

```diff
@@ -0,0 +1,11 @@
+const request = require('request');
+const requestUrl = 'http://localhost:3000/return-status/';
+const interval = 50;
+
+const makeDummyCall = () => setTimeout(() => {
+  const code = 200 + Math.random() * 399;
```

Comment:
minor: what about this in order to retrieve a "real" random http status code?

```javascript
const randomHttpStatusCode = _.sample(_.invert(http.STATUS_CODES));
```

n.b. actually I think I am going to create an npm package that does that 😅